app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'lib/auth-errors',
+  'lib/promise',
+  'stache!templates/partial/coppa-age-input',
+  'views/form',
+  'views/mixins/floating-placeholder-mixin'
+], function (Cocktail, AuthErrors, p, Template, FormView, FloatingPlaceholderMixin) {
+  'use strict';
+
+  var CUTOFF_AGE = 13;
+  var View = FormView.extend({
+
+    template: Template,
+    className: 'coppa-age-input',
+
+    initialize: function (options) {
+      options = options || {};
+
+      this._formPrefill = options.formPrefill;
+    },
+
+    events: {
+      'keydown': 'submitOnEnter',
+      'keyup': 'onInput',
+      'input': 'onInput'
+    },
+
+    /**
+     * Called by the parent view to determine if the user is old enough.
+     * @method isUserOldEnough
+     * @return {Boolean} true if user is old enough, false otw.
+     */
+    isUserOldEnough: function () {
+      return this._getAge() >= CUTOFF_AGE;
+    },
+
+    onInput: function () {
+      // limit age to only 3 characters
+      var age = this.$('#age');
+      age.val(age.val().substr(0, 3));
+    },
+
+    render: function () {
+      var self = this;
+
+      return p()
+        .then(function () {
+          self.$el.html(self.template(self.getContext()));
+          self.initializePlaceholderFields();
+          self._selectPrefillAge();
+        });
+    },
+
+    /**
+     * Called by the parent view to show any validation errors.
+     *
+     * @method showValidationErrorsEnd
+     */
+    showValidationErrorsEnd: function () {
+      if (! this._validateAge()) {
+        this.addInvalidRow('#age');
```

Comment:
I don't think the call to `addInvalidRow` and the `invalid-row` class is needed here, the CSS only specifies a style for `.select-row.invalid-row`:

```bash
➜  styles git:(i2108-coppa-new-v2) ✗ rgrep "invalid-row" .
./main.css:  .select-row.invalid-row {
./modules/_select-row.scss:  &.invalid-row {
./system-font-main.css:  .select-row.invalid-row {
```